test/local/run.sh

```diff
@@ -52,4 +51,23 @@ if [ "$(find ./local -type f | wc -l)" != "1" ]; then
 fi
 pass "Local backups have not been deleted."
 
+# The third part of this test checks if old backups get deleted when the retention
+# is set to 7 days (which it should)
+
+BACKUP_RETENTION_DAYS="7" docker compose up -d
+sleep 5
+
+echo "## Create first backup with no prune"
```

Comment:
In `util.sh` there's an `info` helper for printing additional information with the context of a test. Maybe it'd be helpful to use it here?

Reply:
Didn't realize that existed. Will add that.